### PR TITLE
Vertical alignment

### LINKED ISSUE #130: Redesign: Page vertical realignment
## Alignment
- Desktop - Homepage, About, Jobs index, Jobs post, 404: Distance between bottom of header and top of brandmark should be 144px. 
- Mobile - Homepage, About, Jobs index, Jobs post, 404:  Distance between bottom of header and top of brandmark should be 104px.

### PR DESCRIPTION
**Pull request by machikoyasuda**
requires #134 
closes #130 

## What this PR does
- Adds CSS that declares the following: The first `row` in the `main` element with the `container-fluid` class (`main.container-fluid > .row:first-child`) has a margin-top/bottom of 144px or 104px depending on the device/window width.
- The `default.html` layout has a `<main class="container-fluid">`. So this means that whenever a new page is created, the developer _MUST_ start the page with a `<section class="row">`, in order to get the correct vertical alignment.
- Why not add the top/bottom margins to `<main class="container-fluid">` directly? Because that's not what Figma calls for, and it would create design errors on the About page.

## All new pages that use the Default layout should have

```html
<section class="row">
  <div class="col-lg-7 offset-lg-1">
      # this is element that will get the 144/104px top/bottom margin
  </div>
</section>
```

### CODE REVIEW

File: jobs.html [hunk 1]
```diff
@@ -3,23 +3,22 @@
...
 
-            <p class="my-5">
-                Missed a past opportunity? <a href="http://eepurl.com/h6qTKL">Subscribe</a> to our mailing list, as we
-                are always looking for great candidates.
-            </p>
-        </div>
+        <p class="mt-5 mb-0">
+            Missed a past opportunity? <a href="http://eepurl.com/h6qTKL">Subscribe</a> to our mailing list, as we
+            are always looking for great candidates.
+        </p>
```
Comment (machikoyasuda) [inline jobs.html:21]: Correcting confusing indentation

File: _layouts/job_post.html [hunk 2]
```diff
@@ -4,46 +4,45 @@
...
+                rel="noopener noreferrer">
+                    #banthebox
+                </a>
+            </em>
+        </p>
+
+        <div>
+            <a class="monospace lg-link" href="/jobs">Back to Jobs</a>
+        </div>
+    </div>
```
Comment (machikoyasuda) [inline _layouts/job_post.html:45]: Correcting confusing indentation

**Comment by thekaveman** [comment]
@machikoyasuda 

> All new pages that use the Default layout should have...

Would it make sense to build this into the default layout? So we don't have to repeat this layout code in other pages?

**Comment by machikoyasuda** [comment]
@thekaveman No, because some off-pages like the Blog wouldn't work with it.

**Comment by machikoyasuda** [comment]
@thekaveman I wrote the docs there to just remind people that the page should start with <div class="row"> and then columns within it. It can be easy to forget there's a container already and start the page with <div class="container"> but that would mess up how the column children get rendered.

**Comment by thekaveman** [comment]
@machikoyasuda there is a conflict here, please rebase / resolve when you get a chance.

**Comment by machikoyasuda** [comment]
@thekaveman Rebased + resolved conflicts 🙏

**Comment by machikoyasuda** [comment]
@thekaveman This is ready for review 👋

**Review by thekaveman — COMMENTED** [review]
I think the animation on the About page needs to be adjusted? This is what I'm seeing on the preview site -- the bus is below the bike and the box rather than on top of the box:

image

**Comment by machikoyasuda** [comment]
@thekaveman thanks for the catch! fix is here: https://deploy-preview-135--compilerla.netlify.app/about

**Review by thekaveman — APPROVED** [review]
This looks so much better thank you!! :pray: :+1:

**Pull request merged**